ext/standard/basic_functions.stub.php

```diff
@@ -109,9 +109,9 @@ function min(mixed $arg, mixed ...$args): mixed {}
 
 function max(mixed $arg, mixed ...$args): mixed {}
 
-function array_walk(array|object &$input, callable $funcname, mixed $userdata = UNKNOWN): bool {}
+function array_walk(array|object &$input, callable $callback, mixed $arguments = UNKNOWN): bool {}
```

Comment:
```suggestion
function array_walk(array|object &$input, callable $callback, mixed $argument = UNKNOWN): bool {}
```
Same for the recursive version, as IIRC this argument is passed as the 3rd param to the callback, in which case singular would seem more appropriate.